doc/blameable.md

```diff
@@ -1,614 +1,266 @@
-
-Content:
-
-- [Including](#including-extension) the extension
-- Entity [example](#entity-mapping)
-- Document [example](#document-mapping)
-- [Xml](#xml-mapping) mapping example
-- Advanced usage [examples](#advanced-examples)
-- Using [Traits](#traits)
-
-<a name="including-extension"></a>
-
-## Setup and autoloading
-
-Read the [documentation](./annotations.md#em-setup)
-or check the [example code](../example)
-on how to setup and use the extensions in most optimized way.
-
-<a name="entity-mapping"></a>
+```php
+use Gedmo\Blameable\BlameableListener;
 
-## Blameable Entity example:
+$listener = new BlameableListener();
 
-### Blameable annotations:
-- **@Gedmo\Mapping\Annotation\Blameable** this annotation tells that this column is blameable
-by default it updates this column on update. If column is not a string field or an association
-it will trigger an exception.
+// The $om is either an instance of the ORM's entity manager or the MongoDB ODM's document manager
+$om->getEventManager()->addEventSubscriber($listener);
+```
 
-### Blameable attributes:
-- **\#[Gedmo\Mapping\Annotation\Blameable]** this attribute tells that this column is blameable
-by default it updates this column on update. If column is not a string field or an association
-it will trigger an exception.
+Then, once your application has it available (i.e. after validating the authentication for your user during an HTTP request),
+you can set a reference to the user to be blamed for changes by calling the listener's `setUserValue` method.
 
-Available configuration options:
+```php
+// The $user can be either an object or a string
+$listener->setUserValue($user);
+```
 
-- **on** - is main option and can be **create, update, change** this tells when it
-should be updated
-- **field** - only valid if **on="change"** is specified, tracks property or a list of properties for changes
-- **value** - only valid if **on="change"** is specified and the tracked field is a single field (not an array), if the tracked field has this **value**
-then it updates the blame
+## Configuring Blameable Objects
 
-**Note:** that Blameable interface is not necessary, except in cases there
-you need to identify entity as being Blameable. The metadata is loaded only once then
-cache is activated
+The blameable extension can be configured with [annotations](./annotations.md#blameable-extension),
+[attributes](./attributes.md#blameable-extension), or XML configuration (matching the mapping of
+your domain models). The full configuration for annotations and attributes can be reviewed in
+the linked documentation.
 
-**Note:** these examples are using annotations and attributes for mapping, you should use
-one of them, not both.
+The below examples show the simplest and default configuration for the extension, setting a field
+when the model is updated.
 
-Column is a string field:
+### Annotation Configuration
```

Comment:
As this driver is deprecated, I think we could move this example at the bottom (after Attributes and XML).
```suggestion
### Annotation Configuration (deprecated)
```

Reply:
Updated.